ckanext/canada/tables/choices/classification_group.yaml

```diff
@@ -29,181 +67,482 @@ CT-FIN:
-OE-DEO: {"en": "Office Equipment", "fr": "Mécanographie"}
-OE-MEO: {"en": "Office Equipment", "fr": "Mécanographie"}
-OE-MSE: {"en": "Office Equipment", "fr": "Mécanographie"}
-OM: {"en": "Organization and Methods", "fr": "Organisation et méthodes"}
-OP: {"en": "Occupational and Physical Therapy", "fr": "Ergothérapie et physiothérapie"}
-PC: {"en": "Physical Sciences", "fr": "Sciences physiques"}
-PE: {"en": "Personnel Administration", "fr": "Gestion du personnel"}
-PG: {"en": "Purchasing and Supply", "fr": "Achat et approvisionnement"}
-PH: {"en": "Pharmacy", "fr": "Pharmacie"}
-PI-CGC: {"en": "Primary Products Inspection", "fr": "Inspection des produits primaires"}
-PM: {"en": "Programme Administration", "fr": "Administration des programmes"}
-PO-IMA: {"en": "Police Operations Support", "fr": "Soutien aux opérations policières"}
-PO-TCO: {"en": "Police Operations Support", "fr": "Soutien aux opérations policières"}
-PR: {"en": "Printing Operations", "fr": "Services d'mprimerie"}
-PR-BIN: {"en": "Printing Operations", "fr": "Services d'mprimerie"}
-PR-COM: {"en": "Printing Operations", "fr": "Services d'mprimerie"}
-PR-MAI: {"en": "Printing Operations", "fr": "Services d'mprimerie"}
-PR-OFE: {"en": "Printing Operations", "fr": "Services d'mprimerie"}
-PR-OFO: {"en": "Printing Operations", "fr": "Services d'mprimerie"}
-PR-PRC: {"en": "Printing Operations", "fr": "Services d'mprimerie"}
-PS: {"en": "Psychology", "fr": "Psychologie"}
-PY: {"en": "Photography", "fr": "Photographie"}
-RO: {"en": "Radio Operations", "fr": "Radiotélégraphie"}
-SC-DED: {"en": "Ships' Crews", "fr": "Équipages de navires"}
-SC-EQO: {"en": "Ships' Crews", "fr": "Équipages de navires"}
-SC-ERD: {"en": "Ships' Crews", "fr": "Équipages de navires"}
-SC-SPT: {"en": "Ships' Crews", "fr": "Équipages de navires"}
-SC-STD: {"en": "Ships' Crews", "fr": "Équipages de navires"}
-SE-REM: {"en": "Scientific Research", "fr": "Recherche scientifique"}
-SE-RES: {"en": "Scientific Research", "fr": "Recherche scientifique"}
-SG-PAT: {"en": "Scientific Regulation and Patent Examination", "fr": "Réglementation scientifique et examen des brevets"}
-SG-SRE: {"en": "Scientific Regulation and Patent Examination", "fr": "Réglementation scientifique et examen des brevets"}
+MT:
+  en: Meteorology
+  fr: Météorologie
+ND-ADV:
+  en: Nutrition and Dietetics - Advisory
+  fr: Nutrition et diététique - Conseillers
+ND-DIT:
+  en: Nutrition and Dietetics - Dietitian
+  fr: Nutrition et diététique - Diététiciens
+ND-HME:
+  en: Nutrition and Dietetics - Home Economist
+  fr: Nutrition et diététique - Économistes ménagères
+NU-CHN:
+  en: Nursing - Community Health
+  fr: Sciences infirmières - Infirmières en santé communautaire
+NU-EMA:
+  en: Nursing - Medical Adjudicator Nursing
+  fr: Sciences infirmières - Évaluateurs médicaux
+NU-HOS:
+  en: Nursing - Hospital Nursing
+  fr: Sciences infirmières - Infirmières d’hôpital
+#TODO: find OC group/classification/sub-group
```

Comment:
These changes are all manual? Would be great to add a comment at the top with the data sources used

Reply:
Comment on top of file there with datasource and statement that it is managed manually